README.md

```diff
@@ -1 +1,142 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>HMTL</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **Dynamically generated HTML**
+- **Responsive for both desktop and mobile versions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+You can checkout the running website here:
+
+- [Live Demo Link](https://jakariajishan.github.io/Capstone-1/)
+
+My presentation of the project in its current state:
+
+- [Presentation](https://www.loom.com/share/375f3bfb04f747a685053d7429744cda)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+- VS Code installed.
+- A GitHub account.
+- Git Bash.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- On VS Code, you can click on the "Source Control" icon on the left side bar.
+- From there click on the "Clone Repository" button.
+- Paste the URL from this project and choose a folder to store it.
+- Open the project.
+
+I do recommend this method given it is easier, but if you want you can make this process by running commands on the Git Bash terminal.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👤 Author <a name="author"></a>
+
+Jakaria Jishan
+
+- GitHub: [@Jakaria jishan](https://github.com/JakariaJishan)
+- Twitter: [@Jakaria](https://twitter.com/jakaria_jishan)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Additional functional pages**
+- **User info collection through forms**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/JakariaJishan/Capstone-1/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please make sure to visit my page and checkout my other works as well.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Original design idea by [Cindy Shin in Behance](https://darkxdev.github.io/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)<tra/p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] [optional] Kindly ensure the link to your MIT license is working, currently its broken.